src/GlobalStyles.js

```diff
@@ -5,18 +5,15 @@ const styled = {createGlobalStyle};
 
 export default styled.createGlobalStyle`
   * {
+    margin: 0;
+    padding: 0;
     box-sizing: border-box;
+    color: rgba(0, 0, 0, 0.8);
```

Comment:
I would suggest you  to apply the `margin: 0; padding: 0; color: rgba(0, 0, 0, 0.8) `to the `body { ... / }` 
If you want to read more about the Universal Selector in CSS,  I can recommend this article.